util/storage/influxdb/influxdb.go

```diff
@@ -0,0 +1,121 @@
+package influxdb
+
+import (
+	"time"
+
+	"github.com/bblfsh/performance/util"
+	"github.com/bblfsh/performance/util/storage"
+	"github.com/orourkedd/influxdb1-client/client"
```

Comment:
A nit pick here and below - we usually try to visually separate groups of imports: stdlib, same repository, all the rest.

Reply:
on previous projects we had a rule 1) std 2) all the others :)
Done.